app/lib/app/SecvisogramPage/View.js

```diff
@@ -332,12 +335,18 @@ function View({
     const tracking = formValues.doc.document.tracking
     let prefilledSummary = ''
     let prefilledLegacyVersion = ''
+
     if (tracking?.version && tracking.revision_history?.length) {
-      const majorVersion =
-        typeof tracking.version === 'string'
-          ? parseInt(tracking.version.split('.')[0])
-          : tracking.version
-      if (majorVersion >= 1) {
+      const semverVersion = semver.valid(tracking.version)
+      const initialPublicationVersion = semver.parse('1.0.0')
+
+      // prefill only if Integer versioning OR Semantic version is greater than 1.0.0 (after initial Publication)
+      if (
+        !semverVersion ||
+        (semverVersion &&
+          initialPublicationVersion &&
```

Comment:
why do we need this check here?

Reply:
Good find, changed!
I was looking for this but did not find it in the documentation.